src/main/java/org/spongepowered/common/config/serializer/DataSerializableTypeSerializer.java

```diff
@@ -22,7 +22,7 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-package org.spongepowered.common.config;
+package org.spongepowered.common.config.serializer;
```

Comment:
Nah, there's a better way for handling anything that is registered for `DataTranslator`s, you can use each instance that is being registered to the `SpogneDataManager` and add a new `TypeSerializer` if applicable.